### PR TITLE
Update README.

### PR DESCRIPTION
**Pull request by wanlin31**
This PR is a follow up on #161, I was not able to update that PR further so I opened another one to address additional comments.

### CODE REVIEW

**Review by ybbbby — APPROVED** [review]

File: tools/README.md [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
-scripts are to assist continuous build. The scripts could also help with manual usage
-of prebuilt images if working with GCR. If another image registry is chosen, the prebuilt workers Dockerfiles are available. The root directory of the Dockerfiles could be find [test-infra/containers/pre_built_images](test-infra/containers/pre_built_images).
-
 ## Build and push images
 
-The script [prepare_prebuilt_workers.go](pre_built_workers/prepare_prebuilt_workers.go) 
-builds images and pushes them to a user specified registry. For example, the following shows this process (building and pushing prebuilt images) for cxx and go workers:
+The
+script [prepare_prebuilt_workers.go](pre_built_workers/prepare_prebuilt_workers.go)
+builds images and pushes them to a user specified Google clooud registry. For
```
Comment (paulosjca) [inline tools/README.md:13]: Nit: clooud ➞ cloud.

**Review by paulosjca — APPROVED** [review]

File: tools/README.md [hunk 2]
```diff
@@ -59,10 +73,10 @@ go run test-infra/tools/delete_prebuilt_workers/delete_prebuilt_workers.go \
...
-tag, the tag specified here is removed from image's tag list, otherwise the 
-image is deleted.
+* `-t` <br> Tag for prebuilt images. Tag is the unique identifier for images to
+  delete. If the image have multiple tags including the user specified tag, the
+  tag specified here is removed from image's tag list, otherwise the image is
+  deleted.
 * `-p` <br> Image registry to search images from. Only take image registry
-without the image name. The image registry should also be the most direct 
-registry of images, the directories within image registry will not be checked.
+  without the image name. The image registry should also be the most direct
```
Comment (codeblooded) [inline tools/README.md:81]:
```suggestion
+* `-p` <br> Image registry to search images from. Only accepts the image registry
+  prefix, not the actual image name. If the image registry supports nested repositories,
+ the image registry prefix should be the absolute path to the image's parent repository.
```

**Review by codeblooded — COMMENTED** [review]
Looking good to me, just a few small tweaks to make it read a little better.

File: tools/README.md [hunk 3]
```diff
@@ -22,35 +23,48 @@ go run test-infra/tools/prepare_prebuilt_workers/prepare_prebuilt_workers.go \
...
-[docker tag's restrictions](https://docs.docker.com/engine/reference/commandline/tag/#extended-description). 
+
+* `-l `<br> Language and GITREF to benchmark. The language and its specific
+  GITREF to build the workers from can be specified as `language:COMMIT_SHA`.
+  May be repeated. Valid input for language names are all in lower case:
+  `java`, `go`, `csharp`, `c++`, `cxx`, `node`, `node_purejs`, `ruby`, `php7`
+  , `php7_protobuf_c`, `python` and `python_asyncio`
+* `-t` <br> Tag for prebuilt images. Tag is a required fields. Tag complies with
+  [docker tag's restrictions](https://docs.docker.com/engine/reference/commandline/tag/#extended-description)
+  .
```
[the remark this answers is not in the document]
Comment (wanlin31) [reply tools/README.md:38]: The intelliJ's formatter did it for some reason. :(

**Pull request merged**